nano/templates/safe.tmpl

```diff
@@ -0,0 +1,63 @@
+<head>
+	<style type="text/css">
+		body {
+			background: #3E4E69
+		}
+	</style>
+</head>
+{{if data.open == 0}}
+	<div>
+		<div style='text-align: center;'>
+			<div style='display: inline-block;'>{{:helper.link('Open', 'caret-square-o-down', {'open': 1})}} </div>
+		</div>
+	</div>
+	<div>
+		<div style='text-align: center; margin: 0px 0px 0px 28px'>
+			<div style='display: inline-block;'>{{:helper.link('Left 50', 'arrow-left', {'increment': 50})}} </div>
+			<div style='display: inline-block;'>{{:helper.link('Left 10', 'arrow-left', {'increment': 10})}} </div>
+			<div style='display: inline-block;'>{{:helper.link('Left 1', 'arrow-left', {'increment': 1})}} </div>
+			<div style='display: inline-block;'><div class="link yellowButton"> {{:data.dial}} </div></div>
+			{{if data.unlocked}}
+				<div style='display: inline-block;'>{{:helper.link('Right 1', 'arrow-right', null, 'disabled')}} </div>
+				<div style='display: inline-block;'>{{:helper.link('Right 10', 'arrow-right', null, 'disabled')}} </div>
+				<div style='display: inline-block;'>{{:helper.link('Right 100', 'arrow-right', null, 'disabled')}} </div>
+			{{else}}
+				<div style='display: inline-block;'>{{:helper.link('Right 1', 'arrow-right', {'decrement': 1})}} </div>
+				<div style='display: inline-block;'>{{:helper.link('Right 10', 'arrow-right', {'decrement': 10})}} </div>
+				<div style='display: inline-block;'>{{:helper.link('Right 50', 'arrow-right', {'decrement': 50})}} </div>
+			{{/if}}
+		</div>
+	</div>
+	<div style='text-align: center;'>
+		<span class="fa fa-long-arrow-down fa-2x"></span>
+	</div>
+	<div style='text-align: center; height: 425px;'>
+		<div style='display: inline-block;'> <img src="safe_dial.png" style='transform: rotate({{:data.rotation}}); z-index: -1;'></div>
+	</div>
+	<div style='text-align: center;'>
+		<div style='font-weight: bold;'>How to open your Scarborough Arms - 2 tumbler safe</div> 
```

Comment:
Personally I don't like that the method to open the safe is written directly on the window.